src/chapters/css/rules.rst

```diff
@@ -0,0 +1,67 @@
+CSS Rules
+=========
+
+How to Write a Style Rules
+--------------------------
+
+CSS style rules cover an infinite amount of amount of possibilities.
+Programmers initialize individual CSS properties within the declaration block.
+Every property in CSS has a default value. For example, ``font-color`` defaults to "black".
+Programmers only have to change what needs to be.
+Below are some examples of common CSS properties and what they do.
+It is by no means an exhaustive list of CSS properties, but it is a good place to start.
+
+Here is an example of how to write the declaration block for internal and external CSS:
+
+.. sourcecode:: css
+
+   selector {
+       property: value;
+       property: value;
+       property: value;
+   }
+
+For inline CSS, the declaration block is inside one line of HTML like so:
+
+.. sourcecode:: html
+
+   <tag style="property:value;property:value;property:value;">content</tag>
+
+
+.. admonition:: Note
+
+   HTML elements also default to how it displays on the page.
```

Comment:
The meaning of this note is not clear to me. Does it refer to the default styles/rules that elements have?